non-oss/pharos_pro/addons/kontena-lens/resources/12-ingress.yml.erb

```diff
@@ -10,7 +10,7 @@ spec:
   tls:
     - hosts:
       - <%= host %>
-      <% if config.host %>
+      <% if email %>
```

Comment:
Should we skip whole `tls:` if no email?

Reply:
I assume `https` is not possible with ingress default certificate then